pkg/network/vmicontroller/status.go

```diff
@@ -124,3 +128,11 @@ func calculateSecondaryIfaceStatuses(
 
 	return interfaceStatuses, nil
 }
+
+func filterGuestInternalInterfaces(ifaceStatuses []v1.VirtualMachineInstanceNetworkInterface) []v1.VirtualMachineInstanceNetworkInterface {
```

Comment:
nit: "internal" my imply some subsystem thing, I dont have better suggestion though..
I wonder how we should call interfaces exist in guest but not in spec.

Reply:
Doesn't the term `guest` already suggest an interface that is internal to the guest with its information exposed through the qemu-ga?

> //   - domain.Status.Interfaces: interfaces reported by the guest agent (empty if Qemu agent not running).

I think the `Internal` suffix may be redundant.

Reply:
Continuing the feedback here, I think all will get solved by dropping this wrapper.
If you keep it, then I would express that the function filters interfaces that have been detected from "other" sources. But it is hard to name this.